public/app/core/components/PanelTypeFilter/PanelTypeFilter.tsx

```diff
@@ -2,16 +2,26 @@ import { css } from '@emotion/css';
 import React, { useCallback, useMemo, useState } from 'react';
 
 import { GrafanaTheme2, PanelPluginMeta, SelectableValue } from '@grafana/data';
+import { config } from '@grafana/runtime';
 import { Icon, Button, MultiSelect, useStyles2 } from '@grafana/ui';
-import { getAllPanelPluginMeta } from 'app/features/panel/state/util';
+import { getAllPanelPluginMeta, getVizPluginMeta, getWidgetPluginMeta } from 'app/features/panel/state/util';
 
 export interface Props {
   onChange: (plugins: PanelPluginMeta[]) => void;
   maxMenuHeight?: number;
+  isWidget?: boolean;
 }
 
-export const PanelTypeFilter = ({ onChange: propsOnChange, maxMenuHeight }: Props): JSX.Element => {
-  const plugins = useMemo<PanelPluginMeta[]>(() => getAllPanelPluginMeta(), []);
+export const PanelTypeFilter = ({ onChange: propsOnChange, maxMenuHeight, isWidget = false }: Props): JSX.Element => {
```

Comment:
This component throws when clicking on Import panel (in the empty dashboard state):

```
ReferenceError: isWidget is not defined

    at PanelTypeFilter (webpack-internal:///./public/app/core/components/PanelTypeFilter/PanelTypeFilter.tsx:27:38)
    at div
    at div
    at eval (webpack-internal:///./public/app/features/library-
    ...
```

Reply:
Nice catch 🙌🏾 , I will take a look

Reply:
🤔 I was trying to reproduce this issue, but I could not reproduce it. I tried with different organizations  (with library panels, and without library panels created.)


https://github.com/grafana/grafana/assets/239999/d565cd80-0b1a-4cac-a166-444fca9d9aff

Reply:
Me neither. Could you try again and check if the error persists? @dprokop

Reply:
@juanicabanas @axelavargas - damn, cannot repro this anymore...